examples/sdkconfig.defaults

```diff
@@ -1,5 +1,5 @@
 # Rust often needs a bit of an extra main task stack size compared to C (the default is 3K)
-CONFIG_ESP_MAIN_TASK_STACK_SIZE=24000
+CONFIG_ESP_MAIN_TASK_STACK_SIZE=30000
```

Comment:
Do we still need the increase?

Reply:
verified working with 24000 and on latest commit